how-we-work/all-hands-meeting.html.md

```diff
@@ -1,14 +1,39 @@
 ---
 title: All Hands meeting
 ---
-Every month, we hold an All Hands meeting. These meetings are used to:
-
-- give a general update on the status of Nebulab;
-- introduce new team members;
-- present any important updates to the playbook.
-
-You are strongly encouraged to attend so that you can ask questions and spend some virtual
-bonding time with the entire team, but in case you're not able to, the meetings are recorded.
+Every month, we hold a company-wide meeting, called the All Hands meeting. We use this
+session to provide useful information to every team member, with the intent of being as much
```

Comment:
"to all team members" sounds a bit better in this scenario, because you're providing information to everyone at once, not separately.

Reply:
I think "as much transparent as possible" can be read kind of negatively, as in "there is stuff we don't want to share with you", which is not the case here. Maybe "with the intent of increasing transparency"?